12/umbraco-cms/fundamentals/setup/install/umbraco-on-linux-macos.md

```diff
@@ -21,10 +21,10 @@ With the templates installed on your system, it is now possible to create Umbrac
 To create a project, there are two options:
 
 - Continue creating projects using the .NET CLI.
-- Create new projects using Visual Studio.
+- Create new projects using Visual Studio. (only macOS)
 
 To create new projects using Visual Studio, you can use the [Install using Visual Studio](visual-studio.md) guide.
 
 Once you create a new project it will use SQLite by default on Mac.
 
-If you want to use an SQL server database, you will need to [set up Docker](https://creativewebspecialist.co.uk/2021/09/07/how-to-run-netcore-umbraco-cms-on-a-macbook/) to be able to use SQL Server with Mac.
+If you want to use an SQL server database, you will need to [set up Docker](https://creativewebspecialist.co.uk/2021/09/07/how-to-run-netcore-umbraco-cms-on-a-macbook/) to be able to use SQL Server on Linux or macOS.
```

Comment:
"... to use SQL Server ..." was mentioned twice. Removed one of them! What do you think? Does it still make sense?

```suggestion
If you want to use an SQL server database, you will need to [set up Docker](https://creativewebspecialist.co.uk/2021/09/07/how-to-run-netcore-umbraco-cms-on-a-macbook/).
```